### DISCUSSION TITLE
How to draw Venn diagram & Chemistry Equation like the overleaf

### DISCUSSION THREAD

**Discussion by ghost** [KaTeX/KaTeX#3811] [Q&A]
1. Need to draw venn diagram in Katex
2. Flow chart, design pattern
3. Chemistry equation.

please, teach how to render this kind of text in canvas from start to end. for future generation
5. https://www.overleaf.com/learn/latex/Chemistry_formulae

**Comment by ronkok** [comment]
This is a duplicate of issue #2232. KaTeX does not emulate the `chemfig` package, because it does not emulate the underlying `TikZ` package. (Issue #219).

I think KaTeX is the wrong tool for your task.
